ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -551,6 +551,13 @@ def equal(self, y, name=None):
     @with_unsupported_dtypes({"2.5.1 and below": ("float16", "bfloat16")}, "paddle")
     def rad2deg(self, name=None):
         return paddle_frontend.Tensor(ivy.rad2deg(self._ivy_array))
+    
+    @with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
```

Comment:
You could refer to the supported dtypes in the paddle docs mentioned [here](https://github.com/unifyai/ivy/pull/23702/files#r1335037678)